### PR TITLE
新增tensordot API的中文文档

### PR DESCRIPTION
**Pull request by From00**
中文文档预览链接：http://10.136.157.23:8090/documentation/docs/zh/api/paddle/tensordot_cn.html?reviewVersion=jenkins-doc-review-1424

英文文档预览链接：http://10.136.157.23:8090/documentation/docs/en/api/paddle/tensordot_en.html?reviewVersion=jenkins-doc-review-1424

对应的paddle PR: https://github.com/PaddlePaddle/Paddle/pull/36273

### CODE REVIEW

File: docs/api/paddle/tensordot_cn.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
+.. _cn_api_paddle_tensordot:
+
+tensordot
+-------------------------------
+
+.. py:function:: paddle.tensordot(x, y, axes, name)
```
Comment (zhiqiu) [inline docs/api/paddle/tensordot_cn.rst]: tensordot(x, y, axes=2, name=None)

Comment (From00) [reply docs/api/paddle/tensordot_cn.rst]: done

**Review by zhiqiu — APPROVED** [review]
LGTM

**Review by jzhang533 — APPROVED** [review]
LGTM

**Pull request merged**